packages/mdadm/0001-report-monitor-output-to-syslog.patch

```diff
@@ -0,0 +1,39 @@
+From 06ebc9f352e211cf6adc34a35737da88d2e067c2 Mon Sep 17 00:00:00 2001
+From: Todd Neal <tnealt@amazon.com>
+Date: Fri, 7 Jun 2024 18:22:52 +0000
+Subject: [PATCH] report monitor output to syslog
+
+---
+ systemd/mdmonitor-oneshot.service | 2 +-
+ systemd/mdmonitor.service         | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/systemd/mdmonitor-oneshot.service b/systemd/mdmonitor-oneshot.service
+index ba86b44..032b8aa 100644
+--- a/systemd/mdmonitor-oneshot.service
++++ b/systemd/mdmonitor-oneshot.service
+@@ -10,7 +10,7 @@ Description=Reminder for degraded MD arrays
+ Documentation=man:mdadm(8)
+ 
+ [Service]
+-Environment=MDADM_MONITOR_ARGS=--scan
++Environment=MDADM_MONITOR_ARGS="--scan --syslog"
+ EnvironmentFile=-/run/sysconfig/mdadm
+ ExecStartPre=-/usr/lib/mdadm/mdadm_env.sh
```

Comment:
Seems like we should also patch out the ExecStartPre shell script (which will always fail).

Does it do anything useful?